ui/browser/reducers/main-reducers.js

```diff
@@ -135,3 +142,10 @@ function setCurrentTab(state, pageIndex) {
 function setPageOrder(state, pageOrder) {
   return state.set('pageOrder', pageOrder);
 }
+
+function togglePagesSection(state, opened) {
```

Comment:
I don't think this should handle toggling at all -- either take true or false, and make the state change. Yes we could get states where it's already open, and we try to open it again, but that atleast makes the recorded ACTION more obvious